### DISCUSSION TITLE
Inverted colors in LVGL?

### DISCUSSION THREAD

**Discussion by kchudoba** [witnessmenow/ESP32-Cheap-Yellow-Display#63] [Q&A]
I just set up the CYD with LVGL and in the demos/examples all works but with inverted colors. Any ideas what settings need to be adjusted here?

**Comment by webhooki** [comment]
Did you find solution?

**Reply by tonystarks0113** [reply]
On the User_Setup.h file uncomment one of these lines below:

// #define TFT_INVERSION_ON
// #define TFT_INVERSION_OFF

It works for me and I hope this solution works for you too.

**Comment by iamsako** [comment]
I don't know what LVGL is but I had the same issue with the TFT-eSPI examples. I then noticed that in the main demo examples I tried (the ones in [this folder](https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/tree/main/Examples/Basics)), the colors in the code were also the opposite of what was being displayed. It's a quick hack but adding `tft.invertDisplay(1);` after `tft.init();` fixed it for me. Not sure if that applies to LVGL though.

**Comment by MHz000** [comment]
try this:   tft.invertDisplay(1);       just behind tft.ini() it will invert coloe order

**Comment by TheNitek** [comment]
You probably have the version with the USB-C connector (CYD2USB), so you need to use this config: https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/tree/main/DisplayConfig/CYD2USB

**Comment by clach04** [comment]
@kchudoba can we get this marked as answered? Are you up and running now?
